code/modules/vending/_vending.dm

```diff
@@ -854,7 +854,7 @@ GLOBAL_LIST_EMPTY(vending_products)
 	obj_flags |= EMAGGED
 	to_chat(user, span_notice("You short out the product lock on [src]."))
 
-/obj/machinery/vending/_try_interact(mob/user)
+/obj/machinery/vending/interact(mob/user)
```

Comment:
this is a behavior change. 

https://github.com/tgstation/tgstation/blob/159f01aa893ca7c713773688402c3c0000254fd1/code/game/machinery/_machinery.dm#L709-L714

Reply:
Behavior doesn't change? If you have a vending machine with it's panel open, you can not click on it with a multitool+screwdriver+wirecutters (or just about anything that... isn't screwdrivers) without getting shocked, only being able to access the wires when you put on a pair of insulated gloves. Was there something specific that we're missing here?

Reply:
Oh yeah on second look I had it reversed. This would actually fix technoshy / technointroverts being able to upright the machine.